__tests__/__snapshots__/feeds.ts.snap

```diff
@@ -7,8 +7,8 @@ Object {
     "golang",
   ],
   "excludeSources": Array [
-    "b",
     "c",
+    "b",
```

Comment:
Flaky test?

Reply:
I guess so, my global snapshot update updated this one